src/Controller/Component/FlashComponent.php

```diff
@@ -42,7 +43,9 @@ class FlashComponent extends Component
     protected $_defaultConfig = [
         'key' => 'flash',
         'element' => 'default',
-        'params' => []
+        'params' => [],
+        'reset' => false,
+        'stackLimit' => 50
```

Comment:
Should this be 'limit', like for the paginator options? I know they are unrelated limits but limits nonetheless, what does everything think?